docker/settings.py

```diff
@@ -0,0 +1,120 @@
+from .base_settings import *
+import os
+from django.urls import reverse_lazy
+
+# BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+ALLOWED_HOSTS = ['*']
+
+if os.getenv("ENV") == "localdev":
+    DEBUG = True
+else:
+    DEBUG = False
+
+
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', 'xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx')
+
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'pivot', 'templatetag_handlebars', 'compressor',
+]
+
+INSTALLED_APPS += [
+    'django_prometheus',
+    'django.contrib.humanize',
+    'django_user_agents',
+    # 'supporttools',
+    # 'rc_django',
+]
+
+INSTALLED_APPS += ['uw_saml',]
+
+WEBPACK_LOADER = {
+    'DEFAULT': {
+        'BUNDLE_DIR_NAME': 'pivot/bundles/',
+        'STATS_FILE': os.path.join(BASE_DIR, 'pivot', 'static', 'webpack-stats.json'),
+    }
+}
+
+CSV_ROOT = os.path.join(BASE_DIR, "data/")
+STATIC_ROOT = 'static/'
+COMPRESS_ROOT = 'static/'
+
+
+COMPRESS_PRECOMPILERS = (('text/less', 'lessc {infile} {outfile}'),)
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'compressor.finders.CompressorFinder',
+)
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+]
+
+ROOT_URLCONF = 'project.urls'
+
+TEMPLATES = [
```

Comment:
This one also need to append the context_processors to the one defined in the base_settings